README.md

```diff
@@ -11,82 +15,87 @@ The client offers a number of methods to interact with the API.
 from hundred_x.client import HundredXClient
 from hundred_x.enums import Environment
 
-DEFAULT_SYMBOL="ethperp"
-
 client = HundredXClient(
     private_key="your_private_key",
     environment=Environment.PROD
 )
 
-# Get the current price of a symbol
+# List available products
 products = client.list_products()
 print(products) 
 
 # Get the current price of a symbol
-price = client.get_product(DEFAULT_SYMBOL)
+price = client.get_product("ethperp")
 print(price)
+
+# Place an order
+order = client.place_order(
+    symbol="ethperp",
+    side="BUY",
+    order_type="LIMIT",
+    post_only=False,
+    size="0.1",
+    price="1800",
+    limit_fee_rate="0.001"
+)
+print(order)
 ```
 
-Fopr a demonstration of the async client please refer to the file in examples/async_client.py
+For asynchronous usage, refer to 'examples/async_client.py'.
 
+## Development
 
-## Installation
-```bash
-pip install hundred-x
-```
+### Prequisites
 
-## Running a dockerised environment
-```bash
-docker buildx build --platform linux/amd64 . -t test
-# now we have the dependencies installed,
-# we can mount the current directory and run the tests against the dockerised environment
-docker run -v (pwd):/app -it test
-```
+- Python 3.8+
+- [Poetry](https://python-poetry.org/)
 
+### Setup
 
-## Development
+1. Clone the repository:
 
-### Installation
 
-```bash
-git clone git@github.com:8ball030/hundred_x.git
+```shell
+git clone https://github.com/8ball030/hundred_x.git
 cd hundred_x
-make install
 ```
 
-### Formatting
+2. Install dependencies:
 
-```bash
-make fmt
+```shell
+poetry install
 ```
 
-### Linting
+### Development Commands
 
-```bash
-make lint
-```
+```shell
+# Format Code
+poetry run make fmt
 
-### Tests
+# Lint Code
+poetry run make lint
 
-```bash
-make tests
-```
+# Run Tests
+poetry run make tests
 
-For convience, all commands can be run with:
+# Run all checks
+poetry run make all
 
-```
-make all
+# Create a new release
+poetry run make release
```

Comment:
Better solution here, is using poetry in bash

Then we have a nice small entry point. and in later node projects for example, we can have make install